libusb/descriptor.c

```diff
@@ -84,6 +84,11 @@ static int parse_endpoint(struct libusb_context *ctx,
 	endpoint->wMaxPacketSize = ReadLittleEndian16(&buffer[4]);
 	endpoint->bInterval = buffer[6];
 	if (header->bLength >= LIBUSB_DT_ENDPOINT_AUDIO_SIZE) {
+		if (size < LIBUSB_DT_ENDPOINT_AUDIO_SIZE) {
```

Comment:
We already check on line 74 that size >= header->bLength, so if header->bLength >= x then size must be >= x as well.

Reply:
Yeah, you're right. My previous reasoning seems totally wrong. This commit is therefore not needed.